packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -287,16 +289,16 @@ export class MultiSelectExample extends React.PureComponent<ExampleProps, MultiS
         this.selectFilms([film]);
     }
 
-    private selectFilms(filmsToSelect: Film[]) {
+    private selectFilms(filmsToSelect: readonly Film[]) {
         this.setState(({ createdItems, films, items }) => {
             let nextCreatedItems = createdItems.slice();
             let nextFilms = films.slice();
             let nextItems = items.slice();
 
             filmsToSelect.forEach(film => {
                 const results = maybeAddCreatedFilmToArrays(nextItems, nextCreatedItems, film);
-                nextItems = results.items;
-                nextCreatedItems = results.createdItems;
+                nextItems = results.items.slice();
+                nextCreatedItems = results.createdItems.slice();
```

Comment:
Do we have to slice these? We're not actually mutating either of them below.

Reply:
Ah, yeah, good call. I originally added the slice here since, otherwise, we'd be assigning the readonly arrays from results (`results.items` and `results.createdItems`) to mutable variables, which results in a type error. We can remove the slice altogether and make everything consistently readonly. 5b8c385361d358ba90e18980189eb5ddc683f6b1